Casks/graalvm-ce-java16.rb

```diff
@@ -0,0 +1,48 @@
+cask 'graalvm-ce-java16' do
+  version '21.1.0'
+  sha256 'dafece9d03251304a6d9fc242862cfc08b85e2b8921d3b019a8a19b95af78e2c'
+
+  JVMS_DIR = '/Library/Java/JavaVirtualMachines'.freeze
+  TARGET_DIR = "#{JVMS_DIR}/#{cask}-#{version}".freeze
+
+  url "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-#{version}/#{cask}-darwin-amd64-#{version}.tar.gz"
+  appcast 'https://github.com/oracle/graal/releases.atom'
+  name 'GraalVM Community Edition (Java 16)'
+  homepage 'https://www.graalvm.org/'
+
+  artifact "#{cask}-#{version}", target: TARGET_DIR
+
+  postflight do
+    # Correct symlink 
```

Comment:
for some reason the original symlink points to `/Contents/Home/lib/jli/libjli.dylib` but the actual file is `/Contents/Home/lib/libjli.dylib` (no `jli` subdir)

Reply:
This is a known issue caused by https://bugs.openjdk.java.net/browse/JDK-8235687.
@gilles-duboscq I think you're fixing this (GR-30831)?

Reply:
Without correcting symlink, `xattr` command given below fails with `File not found`, but pass fine with it